kartothek/io_components/metapartition.py

```diff
@@ -622,7 +652,8 @@ def load_dataframes(
             instead of using ``np.datetime64`` to preserve their type. While
             this improves type-safety, this comes at a performance cost. Only
             works for metadata version >= 4.
-
+        predicates: nested list of tuples
+            TODO
```

Comment:
Do we need docs here?

Reply:
Yes, the default docs decorator should work here as well. Not sure about the order of the decorators, though.
my best guess would be:

```
    @_apply_to_list
    @default_docs
    def load_dataframes
```